dev/io.openliberty.jakarta.jstl.2.0/bnd.bnd

```diff
@@ -27,7 +27,7 @@ Import-Package:  jakarta.servlet.jsp.*;version="[2.0.0,4.0.0)", \
  org.apache.taglibs.standard.tag.common.fmt; resolution:=optional, \
  org.xml.sax; resolution:=optional, \
  org.xml.sax.helpers; resolution:=optional, \
- jakarta.el; version="[4.0.0,5.0.0)"
+ jakarta.el; version="[4.0.0,6.0.0)"
```

Comment:
Why are we updating a EE9 bundle?

Reply:
There are tests in OL verify that each feature starts up properly. If I didn't update this section, then faces-4.0 would fail (due to jakarta.el version 5.0 not being found).

I expect us to keep this bundle the same between EE9 and EE10?